### PR TITLE
Javadocs

### PR DESCRIPTION
**Pull request by craftedswift**
PLEASE READ THE JAVADOCS UNDER YOUR USE CASE AND MAKE SURE IT IS CORRECT

### CODE REVIEW

**Review by dozysushi — APPROVED** [review]

**Review by Wizard603 — DISMISSED** [review]
I only had time to look over the ones relating to my use cases. But from what I can see they are accurate.

**Review by dozysushi — DISMISSED** [review]

File: src/main/java/entities/Metric.java [hunk 1]
```diff
@@ -105,6 +104,8 @@ public ArrayList<DataPoint> getDataPoints() {
      * @param date represents the date of the data point
      * @return a boolean value that indicates whether the data point exists
      */
+    // preexistingDataPoint is never used? Should we remove this method??
+    //ASDASDASDASDASDASDASDASDASDASDASDASDASDASD
```
Comment (Bensonwu02) [inline src/main/java/entities/Metric.java:108]: I will deal with this in my upcoming pull request

File: src/main/java/models/DataLoggerResponseModel.java [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 
 /**
- * Class that represents the ResponeModel for DataLogger
+ * Class that represents the ResponseModel for DataLogger
  */
 public class DataLoggerResponseModel{
 
     private final String metricName;
     private final double value;
+    // this variable is never accessed?
```
Comment (Bensonwu02) [inline src/main/java/models/DataLoggerResponseModel.java:10]: This as well

**Review by Bensonwu02 — APPROVED** [review]
I will be adding to your javadocs and fixing the two lines of code I commented on.

**Review by Bensonwu02 — APPROVED** [review]

**Review by Wizard603 — APPROVED** [review]
Docs for Import and Export look good

**Review by tylerkj1209 — APPROVED** [review]

**Pull request merged**